roles/sap_ha_pacemaker_cluster/tasks/construct_vars_nwas_abap_ascs_ers.yml

```diff
@@ -36,7 +36,8 @@
         value: "{{ sap_ha_pacemaker_cluster_nwas_abap_ers_sapinstance_start_profile_string }}"
       - name: AUTOMATIC_RECOVER
         value: "{{ sap_ha_pacemaker_cluster_nwas_abap_ers_sapinstance_automatic_recover_bool | string }}"
-
+      - name: IS_ERS
+        value: true
```

Comment:
This is only required for ENSA1 and is part of the set_fact task that follows this ENSA2 task.
See lines 60-61.

Reply:
Yes and No, it is still recommended configuration for ENSA2 as well, plus it is documented in our documentation for Simple mount.
https://documentation.suse.com/sbp/sap-15/html/SAP-S4HA10-setupguide-simplemount-sle15/index.html#id-implementing-the-cluster > 5.3.4 Configuring resources for the ERS Instance

Let me know and I can try to do os_family conditional as we need to keep it for Suse. @ja9fuchs

Reply:
It's optional and works with both. We can keep it, our documentation does not conflict the setting. :+1: